File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Arm.java
package superstitio.powers.masoOnly;

import superstitio.DataManager;

import static superstitio.cards.general.FuckJob_Card.BodyPart;

public class LostBodyPart_Arm extends LostBodyPart {
    public static final String POWER_ID = DataManager.MakeTextID(LostBodyPart_Arm.class);

    public LostBodyPart_Arm() {
        super(POWER_ID);
    }

    @Override
    public BodyPart[] banedBodyPart() {
        return new BodyPart[]{BodyPart.hand};
    }
}

File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Arm.kt
package superstitio.powers.masoOnly

import superstitio.DataManager
import superstitio.cards.general.FuckJob_Card.BodyPart

class LostBodyPart_Arm : LostBodyPart(POWER_ID) {
    override fun banedBodyPart(): Array<BodyPart?> {
        return arrayOf(BodyPart.hand)
    }

    companion object {
        val POWER_ID: String = DataManager.MakeTextID(LostBodyPart_Arm::class.java)
    }
}
